TiDB Cloud CLI New (#17056)
pingcap/docs: tidb-cloud/cli-reference.md

To use the `ticloud` CLI in your terminal, run `ticloud [command] [subcommand]`. If you are using [TiUP](https://docs.pingcap.com/tidb/stable/tiup-overview), use `tiup cloud [command] [subcommand]` instead.

| Command            | Subcommand                                                               | Description                                                                                              |
|--------------------|--------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------|
| auth               | login, logout                                                            | Login and logout                                                                                         |

> | serverless         | create, delete, describe, list, update, spending-limit, regions, shell   | Manage TiDB Serverless clusters                                                                          |

Reviewer: ```suggestion
| serverless (alias s)   | create, delete, describe, list, update, spending-limit, regions, shell   | Manage TiDB Serverless clusters                                                                          |
```

| serverless branch  | create, delete, describe, list, shell                                    | Manage TiDB Serverless branches                                                                          |

> | serverless import  | cancel, describe, list, start                                            | Manage TiDB Serverless [import](/tidb-cloud/tidb-cloud-migration-overview.md#import-data-from-files-to-tidb-cloud) tasks |

Reviewer: why this link? I think it's not suitable here.

Author: it used to be this link, i will remove the link now

| auth               | login, logout                                                            | Login and logout                                                                                         |
| ai                 | -                                                                        | Chat with TiDB Bot                                                                                       |

> | completion         | bash, fish, powershell, zsh                                              | Generate completion script for specified shell                                                           |

Reviewer 2: Does it need to be listed in the TOC-tidb-cloud.md?

| config             | create, delete, describe, edit, list, set, use                           | Configure user profiles                                                                                  |

> | help               | cluster, completion, config, help, import, project, update               | View help for any command                                                                                |

Reviewer 2: Does it need to be listed in the TOC-tidb-cloud.md?

Author: I think `help` needn't to be listed

To use the `ticloud` CLI in your terminal, run `ticloud [command] [subcommand]`. If you are using [TiUP](https://docs.pingcap.com/tidb/stable/tiup-overview), use `tiup cloud [command] [subcommand]` instead.

| Command           | Subcommand                                                               | Description                                                                                             |
|-------------------|--------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------|
| auth              | login, logout                                                            | Login and logout                                                                                        |

> | serverless (alias s)     | create, delete, describe, list, update, spending-limit, regions, shell   | Manage TiDB Serverless clusters                                                                         |

Reviewer 2: ```suggestion
| serverless (alias: s)     | create, delete, describe, list, update, spending-limit, regions, shell   | Manage TiDB Serverless clusters                                                                         |
```

To use the `ticloud` CLI in your terminal, run `ticloud [command] [subcommand]`. If you are using [TiUP](https://docs.pingcap.com/tidb/stable/tiup-overview), use `tiup cloud [command] [subcommand]` instead.

| Command           | Subcommand                                                               | Description                                                                                             |
|-------------------|--------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------|
| auth              | login, logout                                                            | Login and logout                                                                                        |

> | serverless (alias: s)     | create, delete, describe, list, update, spending-limit, regions, shell   | Manage TiDB Serverless clusters                                                                         |

Reviewer 2: ```suggestion
| serverless (alias: s)     | create, delete, describe, list, update, spending-limit, region, shell   | Manage TiDB Serverless clusters                                                                         |
```

## User profile

For the TiDB Cloud CLI, a user profile is a collection of properties associated with a user, including the profile name, public key, private key and OAuth token. To use TiDB Cloud CLI, you must have a user profile.

> ### Create a user profile with Open API key

Reviewer 2: ```suggestion
### Create a user profile with TiDB Cloud API key
```

## User profile

> For the TiDB Cloud CLI, a user profile is a collection of properties associated with a user, including the profile name, public key, private key and OAuth token. To use TiDB Cloud CLI, you must have a user profile.

Reviewer 2: ```suggestion
For the TiDB Cloud CLI, a user profile is a collection of properties associated with a user, including the profile name, public key, private key, and OAuth token. To use TiDB Cloud CLI, you must have a user profile.
```

> ### Create a user profile with Open API key

Reviewer 2: ```suggestion
### Create a user profile with TiDB Cloud API key
```

The following table lists the global flags for the TiDB Cloud CLI.

| Flag                 | Description                                             | Required | Note                                                                                                             |
|----------------------|---------------------------------------------------------|----------|------------------------------------------------------------------------------------------------------------------|

> | --no-color           | Disables color in output.                               | No       | Only works in non-interactive mode. In interactive mode, disabling color might not work with some UI components. |

Reviewer: add `--debug` flag

|----------------------|---------------------------------------------------------|----------|------------------------------------------------------------------------------------------------------------------|
| --no-color           | Disables color in output.                               | No       | Only works in non-interactive mode. In interactive mode, disabling color might not work with some UI components. |
| -P, --profile string | Specifies the active user profile used in this command. | No       | Works in both non-interactive and interactive modes.                                                             |

> | -D, --debug          | Enable debug mode                                       | No       | Works in both non-interactive and interactive modes.:wq                                                          |

Reviewer 2: ```suggestion
| -D, --debug          | Enable debug mode                                       | No       | Works in both non-interactive and interactive modes.                                                          |
```
